### ISSUE TITLE
[Meds / Tech Debt] Remove `mhv_medications_enable_krames_html_sanitization` feature flag

### ISSUE THREAD

**Issue by andreakuhr** [department-of-veterans-affairs/va.gov-team#132588]
## Context
This work came out of the UX / Accessibility discovery findings. [DSVA Slack thread](https://dsva.slack.com/archives/C059Q28DV99/p1770416590013529) and [findings](https://github.com/department-of-veterans-affairs/va.gov-team/issues/127892#issuecomment-3862728387)

## Background
We tested what the prescription documentation pages would look like with no additional manipulation on our end using raw HTML served from our drug sheet provider, Krames, and found that it does not meet our needs. Therefore, we need to keep our sanitization function in place.

## Description
Deprecate `mhv_medications_enable_krames_html_sanitization` and all logic surrounding it in vets-website and vets-api (in that order).

## Acceptance criteria
- [ ] Feature flag is deprecated from vets-website
- [ ] Feature flag is deprecated from vets-api
- [ ] Code review
